android/ui/src/main/java/com/trafi/ui/RouteSearchTab.kt

```diff
@@ -0,0 +1,152 @@
+package com.trafi.ui
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.trafi.ui.component.internal.RouteSearchTabConstants
+import com.trafi.ui.theme.MaasTheme
+import com.trafi.ui.theme.currentTheme
+import com.trafi.ui.theme.darkColors
+
+@Composable
+private val constants
+    get() = RouteSearchTabConstants(currentTheme)
+
+data class TabItem(
+    val id: String,
+    val icon: String,
+    val active: Boolean,
+    val duration: String?,
+    val price: String?,
+)
+
+@Composable
+fun RouteSearchTab(
+    tabItem: TabItem,
+    modifier: Modifier,
+    isTabSelected: Boolean,
```

Comment:
`isSelected` would be a simpler name 🙂 We're already in the context of "route search tab"